tests/spec/fs.appendFile.spec.js

```diff
@@ -110,11 +110,22 @@ describe('fs.appendFile', function() {
     });
   });
 
-  it('should work when file does not exist, and create the file', function(done) {
+  it('should error when the symbolic link does not exist', function(done) {
+    var fs = util.fs();
+     
+    fs.symlink('/myfile', ' ', function(error) {  
+      expect(error).to.exist; 
+      expect(error.code).to.equal('EINVAL');
+      done();  
+    });
+  });
+
+
+  it('should work when file does not exist, and create the file',function(done) {
     var fs = util.fs();
     var contents = 'This is a file.';
 
-    fs.appendFile('/newfile', contents, { encoding: 'utf8' }, function(error) {
+    fs.appendFile('/newfile', contents, { encoding: 'utf8' },function(error) {
```

Comment:
Be careful with changes like this one, which only change whitespace.  Projects tend to not want them, since they end up hiding history in git (i.e., it looks like you altered this file, but really, the code was introduced before you touched this line).